### PR TITLE
Remove disqus and add new blog footer

### PR DESCRIPTION
**Pull request by zankavrogin**

### CODE REVIEW

**Comment by deadprogram** [comment]
Is this ready for merge?

**Comment by zankavrogin** [comment]
It's ready to merge, unless you have any feedback on the new blog footer.

File: source/index.html.haml [hunk 1]
```diff
@@ -144,8 +144,10 @@ getting_started_button: "<a href='/documentation/getting-started' class='call-to
   :markdown
     ## Be Part of the Robot Evolution
```
Comment (deadprogram) [inline source/index.html.haml]: The order of this should be: Twitter, google groups, github, github issues, irc

**Comment by zankavrogin** [comment]
I updated the homepage with the new list order.

**Comment by deadprogram** [comment]
:footprints:

**Pull request merged**